scripts/jenkins/cloud/ansible/roles/input_model_generator/README.md

```diff
@@ -39,6 +39,16 @@ Example:
       -e virt_config_file=/path/to/virt-config.yml
 ```
 
+NOTE: optionally, provide a `cloudsource` value (defaults to `stagingcloud9`) to control which version specific services
+and features are activated and an `ardana_env` value (defaults to `localhost`) to change how the generated items are named:
```

Comment:
you probably mean `cloud_env` instad of `ardana_env`

Reply:
fixed